frontend/src/views/RouteFormView.vue

```diff
@@ -147,21 +144,32 @@ export default {
     ...mapActions(namespaces.passengers, [apiActions.fetchPassengers]),
     handleSubmit() {
       this.validateForm();
-      this.isSubmitted = true;
 
-      if (!this.errors.length) {
-        this.route.id = uuidv4();
+      if (!Object.keys(this.errors).length) {
         this[actions.SUBMIT]({ form: this.route });
         this.route = { ...defaultFormState };
-        this.isSubmitted = false;
       }
     },
 
     validateForm() {
       const { route } = this;
+
+      const makeErrorsPartial = makeErrors(this.$t.bind(this));
+
       this.errors = Object.keys(route)
         .filter(isErroring(route))
-        .map(makeErrorMessage(this.$t.bind(this)));
+        .reduce(makeErrorsPartial, {});
+
+      const { startMileage, endMileage } = this.route;
+
+      if (
+        !!startMileage
+        && !!endMileage
+        && parseInt(startMileage, 10) >= parseInt(endMileage, 10)
+      ) {
+        this.errors.startMileage = this.$t('common.start_mileage_error');
```

Comment:
There should be a separate error for a case when startMileage >= endMileage. It is confusing to see messages "Start mileage is incorrect" and "End mileage is incorrect" in such case.

Reply:
I did consider that, but making error message that is way too specific might be just hard to read and confusing. Let's just make sure that the values are correct and user knows about incorrect ones.